diff --git a/rust/fp.rs b/rust/fp.rs
index d6d4785..54dcdd6 100644
--- a/rust/fp.rs
+++ b/rust/fp.rs
@@ -68,7 +68,7 @@ pub const OMASK: Chunk = (-1) << (MODBITS % big::BASEBITS);
 pub const TBITS: usize = MODBITS % big::BASEBITS; // Number of active bits in top word
 pub const TMASK: Chunk = (1 << TBITS) - 1;

-pub const BIG_ENDIAN_SIGN: bool = false;
+pub const BIG_ENDIAN_SIGN: bool = true;

 impl FP {
     /* Constructors */
